src/libcrun/utils.h

```diff
@@ -127,7 +127,7 @@ make_libcrun_fd_map (size_t len)
   struct libcrun_fd_map *ret;
   size_t i;
 
-  ret = xmalloc (sizeof (*ret) + sizeof (int) * len);
+  ret = (struct libcrun_fd_map *) xmalloc (sizeof (*ret) + sizeof (int) * len);
```

Comment:
@aleks-f What error do we get from `g++` or the compiler which you are using ?

My perception is that `void *` is automatically and safely promoted to any other pointer type. So why do need explicit cast here ? Isn't `ret` already `libcrun_fd_map *`

Reply:
@flouthoc C++ won't implicitly cast from `void*` 

g++  8.3.0 gives this error:
```
crun/utils.h:102:25: error: invalid conversion from ‘void*’ to ‘int*’ [-fpermissive]
   int *pp = /*(int *)*/ p;
```
Adding `-fpermissive` makes it compile, but gives a bunch of these:
```
warning: invalid conversion from ‘void*’ to ‘int*’ [-fpermissive]
   int *pp = /*(int *)*/ p;
```
That's as far as it goes, AFAIK - not a blocker, but life would be better without the ugly noise; there's potentially a small performance price, but I hope gcc can optimize it away.